cmd/reloader/main.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+
+	"flag"
+	"fmt"
+	"log"
+	"log/slog"
+	"net/http"
+	"time"
+
+	"github.com/openperouter/openperouter/internal/frrconfig"
+	"github.com/openperouter/openperouter/internal/logging"
+)
+
+var frrConfigPath string
+
+func main() {
+	var bindAddress string
+	var logLevel string
+	flag.StringVar(&bindAddress, "bindaddress", "0.0.0.0:9080", "The address the reloader endpoint binds to. ")
```

Comment:
can the default be localhost (or we intend this to be public)?

Reply:
the reloader process is going to be part of the frr pod, which is going to be network namespaced, and as such shouldn't be accessible from outside. We need the controller pod to be able to talk to it, and I am not sure localhost would work in this case.